subsys/bluetooth/controller/Kconfig

```diff
@@ -286,5 +286,12 @@ config BT_CTLR_SDC_CX_ADV_CLOSE_ADV_EVT_ON_DENIAL
 endchoice
 endif # MPSL_CX && BT_BROADCASTER
 
+config BT_CTLR_LE_POWER_CONTROL
```

Comment:
Did you consider adding a separate config for LE Path Loss Monitoring? It is listed as a separate feature, so I guess that makes sense

Reply:
Yes that is true but path loss monitoring commands are not fully implemented yet. So I excluded any path loss monitoring related stuff.

Reply:
Just noticed that you added handlers for the the commands. So you could consider wrapping those in another ifdef